examples/energy-monitor/Cargo.toml

```diff
@@ -11,9 +11,11 @@ publish = false
 license = "MIT"
 
 [[bin]]
-path = "src/main.rs"
 name = "energy-monitor"
 
+[lib]
+crate-type = ["cdylib", "lib"]
```

Comment:
I didn't add a name but the default name for the library ("energy_monitor") is not the same as the binary name ("energy-monitor")  and therefore it shouldn't conflict.